src/foam/u2/PropertyModal.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
This seems exactly like the work @RachaelDing and @olhabn were working on.

Reply:
I think we were working on deactivate account modal